src/views/Project/index.jsx

```diff
@@ -1,111 +1,102 @@
-    tag: 'JS',
-    assignee: 'None',
-    lastUpdated: '2018-05-06',
-  },
-  {
-    project: 'Taskcluster',
-    summary: '1451548 - Return 404 for indexes and namespaces that are expired',
-    tag: 'JS',
-    assignee: 'None',
-    lastUpdated: '2018-05-04',
-  },
-  {
-    project: 'Taskcluster',
-    summary: '1441977 - Run tests for taskcluster-treeherder in Taskcluster',
-    tag: 'JS',
-    assignee: 'None',
-    lastUpdated: '2018-05-03',
-  },
-  {
-    project: 'Servo',
-    summary: '1446768 - Only post "No taskcluster jobs.." to a PR once',
-    tag: 'JS',
-    assignee: 'None',
-    lastUpdated: '2018-05-01',
-  },
-  {
-    project: 'Taskcluster',
-    summary:
-      '1457126 - Authorization failures should state which clientId lacks scopes',
-    tag: 'Rust',
-    assignee: 'None',
-    lastUpdated: '2018-04-05',
-  },
-  {
-    project: 'Taskcluster',
-    summary: '44 - Add pagination to auth.listRoles',
-    tag: 'JS',
-    assignee: 'None',
-    lastUpdated: '2018-03-05',
-  },
-  {
-    project: 'Taskcluster',
-    summary:
-      '1306494 - Add a diff+commit submit button to some text areas in tc-tools',
-    tag: 'JS',
-    assignee: 'None',
-    lastUpdated: '2018-02-12',
-  },
-  {
-    project: 'Taskcluster',
-    summary:
-      '333 - Make azure-entities and azure-blob0-storage independent of tc-lib-monitor',
-    tag: 'JS',
-    assignee: 'None',
-    lastUpdated: '2018-02-04',
-  },
-  {
-    project: 'Taskcluster',
-    summary: '1344912 - Support tag events, too',
-    tag: 'JS',
-    assignee: 'None',
-    lastUpdated: '2018-01-01',
-  },
-  {
-    project: 'Taskcluster',
-    summary: '1453714 - Return http 424 instead of 403 for error artifacts',
-    tag: 'Python',
-    assignee: 'None',
-    lastUpdated: '2017-12-04',
-  },
-];
+const githubQuery = fileName => {
+  const projectInfo = projects[fileName];
+  // dictionary with repo as key and tag as value
+  const tmp = projectInfo.repositories
+    ? projectInfo.repositories.reduce((prev, cur) => ({ ...prev, ...cur }), {})
+    : {};
+  // initial query search for all repo
+  const repoQuerySearch = ['state:open'];
+  // a dictionary of tag (string) and list of repository in that tag
+  const tagRepoDictionary = Object.keys(tmp).reduce((prev, key) => {
+    const curr = [
+      ...(prev[tmp[key]] || [...repoQuerySearch, `label:${tmp[key]}`]),
+      `repo:${key}`,
+    ];
+
+    return {
+      ...prev,
+      [tmp[key]]: curr,
+    };
+  }, {});
+  const tagRepoList = Object.entries(tagRepoDictionary).map(([key, value]) => ({
+    label: key,
+    query: value.join(' '),
+  }));
+
+  if (tagRepoList.length === 0) return;
+
+  let allQuery = '';
+
+  tagRepoList.forEach((tag, idx) => {
+    const noCursorQuery = `_${idx}: search(first:100, type:ISSUE, query:"${
+      tag.query
+    }")\n
+      {
+      ...Issues
+      }\n`;
+
+    allQuery = allQuery.concat(noCursorQuery);
+  });
+
+  return allQuery;
```

Comment:
Can you paste a sample query string of what this might return?

Reply:
```
{_0: search(first:100, type:ISSUE, query:"state:open repo:armenzg/mozilla_ci_tools repo:automatedtester/automation-services-bot repo:automatedtester/powerball-platform repo:automatedtester/unittest-zero repo:davehunt/flynnid repo:davehunt/pytest-mozwebqa repo:mozilla/bidpom repo:mozilla/coversheet repo:mozilla/memchaser repo:mozilla/moz-grid-config repo:mozilla/mozdownload repo:mozilla/mozmill-ci repo:mozilla/nightlytt label:mentored")
      {
      ...Issues
      }
_1: search(first:100, type:ISSUE, query:"state:open repo:marco-c/grcov repo:marco-c/code-coverage-reports repo:marco-c/code-coverage-addon label:good-first-bug")
      {
      ...Issues
      }
_2: search(first:100, type:ISSUE, query:"state:open repo:mozilla-releng/services label:skill:good-first-bug")
      {
      ...Issues
      }
}

fragment Issues on SearchResultItemConnection {
  	issueCount
    nodes{
      ...on Issue  {
        title
        updatedAt
        number
        repository {
          name
        }
        url
        labels(last:2){
          nodes{
            name
            color
          }
        }
        assignees(last:1) {
          nodes{
            login
          }
        }
      }
    }
}
```

Reply:
i mean this: 
```
{_0: search(first:100, type:ISSUE, query:"state:open repo:armenzg/mozilla_ci_tools repo:automatedtester/automation-services-bot repo:automatedtester/powerball-platform repo:automatedtester/unittest-zero repo:davehunt/flynnid repo:davehunt/pytest-mozwebqa repo:mozilla/bidpom repo:mozilla/coversheet repo:mozilla/memchaser repo:mozilla/moz-grid-config repo:mozilla/mozdownload repo:mozilla/mozmill-ci repo:mozilla/nightlytt label:mentored")
      {
      ...Issues
      }
_1: search(first:100, type:ISSUE, query:"state:open repo:marco-c/grcov repo:marco-c/code-coverage-reports repo:marco-c/code-coverage-addon label:good-first-bug")
      {
      ...Issues
      }
_2: search(first:100, type:ISSUE, query:"state:open repo:mozilla-releng/services label:skill:good-first-bug")
      {
      ...Issues
      }
}
```
then i add with the fragment in the Query component